dcos/dcos-ui
# DCOS-56348: Quota Overview - Limit Column

Add a limit column for Quota.

Closes https://jira.mesosphere.com/browse/DCOS-56348

## Testing
0. Start localhost and login.
1. Update your `Config.dev.ts`
```
uiConfiguration: {
...
  features: {
    quota: true
  }
},
  useFixtures: true,
  ...
```
2. Go to services page.
3. Open quota tab.
4. Verify that the new "Quota Limit" column is there and it looks and behaves as expected.
5. Verify that the added tests make sense.

## Trade-offs
Some of our solutions will probably need to be changed.

## Dependencies
None.

## Screenshots
![Screenshot from 2019-07-11 16-51-54](https://user-images.githubusercontent.com/40791275/61056592-4ff43000-a3fc-11e9-8bd3-82b9db8b248a.png)
